server/src/main/java/org/kaazing/gateway/server/config/parse/GatewayConfigParser.java

```diff
@@ -412,7 +379,13 @@ private void validateGatewayConfig(GatewayConfigDocument configDoc, List<String>
                         LOGGER.error("  Line: " + line + " Column: " + column);
                     }
                 }
-                LOGGER.error("  " + error.getMessage().replaceAll("@" + GATEWAY_CONFIG_NS, ""));
+                LOGGER.error("  " + error.getMessage().replaceAll("@" + GatewayConfigNamespace.CURRENT_NS, ""));
+                if (error.getMessage().contains("notify-options")) {
```

Comment:
Presumably it could also be [notify element](http://developer.kaazing.com/documentation/jms/4.0/admin-reference/r_conf_service.html#notifyele)